### PR TITLE
Built and checked in cmapi.jar to remove dependency on DI2E.

### PR DESCRIPTION
**Pull request by rsupnekar**

### CODE REVIEW

File: mapengine/worldwind/aar/build.gradle [hunk 1]
```diff
@@ -3,6 +3,7 @@ apply from: "$rootDir/build-support/emp3-gradle-plugin/src/main/resources/androi
 dependencies {
     compile          ("com.android.support:support-annotations")
     provided project (":emp3-android-sdk")
+    provided files ("$rootDir/3rdpartylibrary/cmapi/cmapi.jar")
```
Comment (jrummel) [inline mapengine/worldwind/aar/build.gradle:6]: Could be worth putting a version number on the cmapi jar? Other than that everything looks good to me, approving.

**Review by jrummel — APPROVED** [review]
Could be worth putting a version number on the cmapi jar, approving anyway.

File: mapengine/worldwind/aar/build.gradle (hunk 1 above)
Comment (rsupnekar) [reply mapengine/worldwind/aar/build.gradle:6]: I thought about it.  The version is 0.0.7.   We're still waiting for something like a 1.0

**Comment by rsupnekar** [comment]
This is a build issue, nothing for the tester to test.

**Pull request merged**